src/event/cloudevent.ts

```diff
@@ -167,6 +168,17 @@ export class CloudEvent implements CloudEventV1, CloudEventV03 {
     }
   }
 
+  /**
+   * Emit this CloudEvent through the application
+   *
+   * @param {boolean} ensureDelivery fail the promise if one listener fail
+   * @return {Promise<CloudEvent>} current CloudEvent object
```

Comment:
Nit, but more direct?
```suggestion
   * @return {Promise<CloudEvent>} this
```